core/app/assets/stylesheets/refinery/global/_colours.scss

```diff
@@ -1,18 +1,21 @@
 $icon_default_colour: #404650;
-$icon_page_colour:    #86cffa;   //
-$icon_folder_colour:  #e6cb47;   // dark yellow
-$icon_add_colour: $icon_default_colour;
-// $icon_add_colour:     #7fff00;   // chartreuse
 $icon_delete_colour:  #ff707f;   // rose
-$icon_edit_colour: $icon_default_colour;
-// $icon_edit_colour:    #007fff;   // azure
+$icon_done_colour:    #00a017;  //
 $icon_error_colour:   #ff0;     //RED
-$icon_preview_colour: $icon_default_colour;
-// $icon_preview_colour: #7f00ff;   // violet
+$icon_folder_colour:  #e6cb47;   // dark yellow
 $icon_locale_colour:  #b7e3fc;    // pale blue
-$icon_done_colour:    #00a017;  //
+$icon_page_colour:    #86cffa;   //
+$icon_warning_colour: #FF6600; // orange
+
 $info_icon_blue:      #316CC8;
-$icon_warning_colour:   #FF6600; // orange
+
+$icon_preview_colour: $icon_default_colour;
+$icon_edit_colour: $icon_default_colour;
+$icon_add_colour: $icon_default_colour;
 
 $action_background_colour: lighten($icon_locale_colour,10%);
 $action_border_colour: $icon_locale_colour;
+
+// $icon_preview_colour: #7f00ff;   // violet
+// $icon_edit_colour:    #007fff;   // azure
+// $icon_add_colour:     #7fff00;   // chartreuse
```

Comment:
I could do a complete refactor of the current CSS for this PR but i think it's better to do this in an second PR in order to keep easy the review.